scripts/globals/battlefield.lua

```diff
@@ -83,7 +84,7 @@ function dsp.battlefield.onBattlefieldTick(battlefield, timeinside, players)
                 canLeave = true
             end
         end
-        if canLeave and cutsceneTimer >= 15 then
+        if canLeave and cutsceneTimer >= 15 and not dontKick then
```

Comment:
`local dontKick = battlefield:getLocalVar("dontKick")`
` if canLeave and cutsceneTimer >= 15 and dontKick == 0 then` 
seems saner to me